meeshkan/meeshkan
# Fix duplicate toasters bug

- Clicking the "delete story" and "create test case" buttons (in the individual user story page) multiple times causes duplicate toasts to show up and results in a silent error because the delete mutation is called multiple times.

Testing instructions:
- Try deleting a user story (from both possible buttons).
- Try promoting a user story to a test case.
- Make sure toasts throughout the webapp work.
